azerothcore/azerothcore-wotlk
# Remove requirement for cooking on Beer Basted Boar Ribs quest

## Changes Proposed:
Modify quest_template_addon for quest 384; remove cooking training requirement

## Issues Addressed:

https://github.com/azerothcore/azerothcore-wotlk/issues/14033

## SOURCE:
n/a

## Tests Performed:
Verified quest was available at level 5 with no cooking trained


## How to Test the Changes:
1. Create dwarf
2. .levelup 4
3. .go c id 1267
4. Verify quest is available to give out

## Known Issues and TODO List:
n/a


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
